docs/documentation.md

```diff
@@ -9,33 +9,49 @@ All request should have in URL `stationId` param encoded with Base64.
 **Tasks:**
 
 - `GET /tasks` - Get all tasks for the authenticated user.
+    - Request
```

Comment:
:bulb: Idea for the future, it could be added in next PR, but I think it could be good to create simple docs, how to run backend and what ports are used to access it

Reply:
I will wrap backend into docker image and prepare mentioned doc but in next PR ( This one is big enough )

Reply:
Sure, good idea!